scripts/parse.py

```diff
@@ -18,13 +18,17 @@
 # along with taserver.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+from typing import Optional, Set
```

Comment:
This is the first time I see the typing module. Can you tell me why I should be using this in all my future python code? =)

Reply:
Downsides of typing: it only works on Python >=3.5.

But upsides: aside from being explicit and having good integration with some IDEs / editors, the main benefit is you can use http://mypy-lang.org/ to statically typecheck your Python code (Mypy works really well tbh).